content-services/content-sources-frontend
# App: Hide global filter bar

## Summary
Hide the global filter that really only should apply to pages in Insights that have an inventory and support tagging.

This change is borrowed from Image Builder's frontend: https://github.com/osbuild/image-builder-frontend/blob/4a893e10baa0500136e9be98b15645d908b8f0b7/src/App.tsx#L14

## Testing steps

1. Open the 'Repositories' page.
2. Verify that the global filter at the top is gone.
